### ISSUE TITLE
populate/deepPopulate for all content types by default

### ISSUE THREAD

**Issue by gkarmas** [strapi/strapi#1980]
This is a feature request started from [https://github.com/strapi/strapi/issues/1783](this) and [https://github.com/strapi/strapi/issues/1886#issuecomment-418291091](this) tickets.
Simple object without using graphql on strapi, does not return anything else but string objects.
![45195289-c6486480-b224-11e8-9fb5-d8a4ef9eb498](https://user-images.githubusercontent.com/8575682/45722138-fe329e80-bb78-11e8-9912-13ee438df47f.png)

**Comment by gkarmas** [comment]
@Aurelsicoko no. Playground and strapi/users/, are displaying all levels. Object return doesn't for users. Direct api calls don't return any sub-levels of the users property but "role". Everything else is ignored. 

![user 2018-10-01 21-44-03](https://user-images.githubusercontent.com/8575682/46324782-3eefd600-c5c3-11e8-9585-72208cb8e461.png)
![graphql 2018-10-01 21-46-49](https://user-images.githubusercontent.com/8575682/46324849-942be780-c5c3-11e8-815b-8940554f3f0c.png)

**Comment by Aurelsicoko** [comment]
@kamalbennani Any idea? It makes no sense, is this related to Strapi or Apollo?

**Comment by gkarmas** [comment]
@Aurelsicoko  this is still unresolved. Did you confirm ? **Users -> DeepPopulate** is still locked and no way to deep populate any custom fields.

**Comment by Aurelsicoko** [comment]
It seems that no one has this issue. Can you try to identify why this is not working as excepted in your case?

**Comment by phr3nzy** [comment]
I've been having this issue for a while now and have been scratching my head all day.

**Comment by Erim32** [comment]
@Aurelsicoko this is still unresolved, bug present in v3.0.0-beta 14.
To reproduce this issue:
1 - just start a new strapi project
2 - add media field to User model
3 - create an entry and upload a file project form /admin
4 - compare login response using postman POST with ( /auth/local ) and get form /api/users

Auth login fetching seems ingore File Upload entries
